src/lib/Http/FormatAttributeHtmlFormatRequestMatcher.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace EzSystems\HybridPlatformUi\Http;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestMatcher;
+
```

Comment:
Well, it may sound like it, but "format" is a noun in that case (the "format" request attribute). Do you have an alternative in mind.